changelog/unreleased/pull-2906

```diff
@@ -0,0 +1,11 @@
+Bugfix: Restore timestamps and permissions on intermediate directories
+
+When using the `--include` option with the restore command, it only restored
+timestamps and permissions on directories selected by the include pattern.
+Intermediate directories which are necessary to restore files located in
+subdirectories, were created with default permissions. We've fixed the
+`restore` command to also restore permissions for these directories.
```

Comment:
I suggest a small change to this:

~~~
When using the `--include` option of the restore command, restic restored
timestamps and permissions only on directories selected by the include pattern.
Intermediate directories, which are necessary to restore files located in sub-
directories, were created with default permissions. We've fixed the `restore`
command to restore timestamps and permissions for these directories as well.
~~~

Reply:
Modified